databases/week3/Additional_queries.sql

```diff
@@ -0,0 +1,49 @@
+
+-- Get meals that has a price smaller than a specific price fx 90
+use mealsharing;
+SELECT * from meal
+where price  < 90;
+
+-- Get meals that still has available reservations
+select meal.id, max_reservations , sum(max_reservations - number_of_guests) as available_reservations
+from meal
+inner join reservation 
+on  meal.id = reservation.meal_id;
+
+-- Get meals that partially match a title.
+ -- Rød grød med will match the meal with the title Rød grød med fløde
+ 
+ select * from meal 
+ where title like '% alfredo%';
+ 
+ -- Get meals that has been created between two dates
+  select * from meal 
+ where DATE(created_date) between '2021-04-30' AND '2021-05-01' ;
+ 
+ -- Get only specific number of meals fx return only 5 meals
+   select * from meal 
+   limit 2;
+   
+   -- Get the meals that have good reviews
+select meal.id , meal.title ,  review.stars
+from meal 
+inner join review
+On meal.id = review.meal_id
+where review.stars > 3 ;
+
+-- Get reservations for a specific meal sorted by created_date
```

Comment:
Hummm, i think instead of getting all the reservations in a specific month you should be getting all the reservations for a specific meal, for instance meal with id=3.